pkg/storaged/stratis/create-dialog.jsx

```diff
@@ -28,6 +28,44 @@ import { validate_url, get_tang_adv } from "../crypto/tang.jsx";
 
 const _ = cockpit.gettext;
 
+function manager_create_pool(name, devs, key_desc, clevis_info) {
+    // "clevis_info" is either falsy or an array with two elements:
+    //
+    //   [ pin, pin_config ]
+    //
+    // This array can be used directly with the (b(ss)) signature of
+    // the r6 CreatePool call, but needs to be taken apart for the
+    // ((bu)ss) signature of the r8 CreatePool call.
+
+    if (client.stratis_interface_revision < 8) {
+        let key_desc_arg = [false, ""];
+        if (key_desc)
+            key_desc_arg = [true, key_desc];
+        let clevis_info_arg = [false, ["", ""]];
+        if (clevis_info)
+            clevis_info_arg = [true, clevis_info];
+        return client.stratis_manager.CreatePool(name,
+                                                 devs,
+                                                 key_desc_arg,
+                                                 clevis_info_arg);
```

Comment:
These 11 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-21927-4a5fe6bc-20250826-110716-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)